src/common/timing/start-time.js

```diff
@@ -6,31 +6,29 @@
 // Use various techniques to determine the time at which this page started and whether to capture navigation timing information
 
 import { mark } from './stopwatch'
-import { setOffset } from './now'
+import { setOffset, getLastTimestamp } from './now'
 import { globalScope } from '../util/global-scope'
 
 export let navCookie = true
 
-// findStartTime() *cli - comment out to add agentId context which is n/a at import time
-
 export function findStartTime (agentId) {
-  // findStartCookie is used for FF7/8 & browsers which do not support window.performance.timing.navigationStart
-  var starttime = findStartWebTiming() // || findStartCookie() -- now redundant *cli oct'22, TO DO: slated for removal
+  var starttime = findStartWebTiming()
   if (!starttime) return
 
   mark(agentId, 'starttime', starttime)
   setOffset(starttime)
+  return starttime
 }
 
 // Find the start time from the Web Timing 'performance' object.
-// The "timeOrigin" property is the new standard timestamp property shared across main frame and workers, but is not supported in some early Safari browsers (safari<15)
+// The "timeOrigin" property is the new standard timestamp property shared across main frame and workers, but is not supported in some early Safari browsers (safari<15) + IE
 // navigationStart is deprecated, but can still be used as a fallback
 // http://test.w3.org/webperf/specs/NavigationTiming/
 // http://blog.chromium.org/2010/07/do-you-know-how-slow-your-web-page-is.html
 function findStartWebTiming () {
   // note that we don't need to use a cookie to record navigation start time
   navCookie = false
   const timestamp = globalScope?.performance?.timeOrigin || globalScope?.performance?.timing?.navigationStart
-  if (!timestamp) return
-  return Math.round(timestamp)
+  if (!timestamp) return getLastTimestamp()
```

Comment:
Should the result of `getLastTimestamp` also go through `Math.floor`?

Reply:
getLastTimestamp uses `new Date().getTime()` at its core, which only returns integers.  But for stability, I think it might be wise to just ensure that all these numbers get rounded.  Good call, will add a commit